microsoft-edge/devtools-guide-chromium/whats-new/2021/10/devtools.md

```diff
@@ -0,0 +1,108 @@
+---
+# What's New in DevTools (Microsoft Edge 95)
+
+[!INCLUDE [note about What's New announcements from the Microsoft Edge DevTools team](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Sync live changes from the Styles tool by using CSS Mirror Editing
+
+<!-- Title: CSS Mirror Editing in VS Code -->
+<!-- Subtitle: Changes in the CSS Styles editor now show up in your CSS files inside Visual Studio Code. -->
+
+The Microsoft Edge DevTools extension for Visual Studio Code now features an experiment called **CSS Mirror Editing**.  Using this feature, you can use the **Styles** tool to adjust your CSS, and the changes you apply are automatically added to the CSS source files in the editor.  You can toggle this functionality on and off.
+
+For more information, see [Microsoft Edge DevTools extension for Visual Studio Code](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#syncing-live-changes-from-the-styles-tool-by-using-css-mirror-editing).
+
+To provide feedback, in Visual Studio Code, in the **Activity Bar**, click **Microsoft Edge Tools**, and then in the **CSS Mirror Editing** section, click the **leaving feedback here** link.
+
+:::image type="content" source="../../media/2021/10/css-mirror-editing-button.msft.png" alt-text="Changes in the CSS Styles editor now show up in your CSS files inside Visual Studio Code." lightbox="../../media/2021/10/css-mirror-editing-button.msft.png":::
+
+
+<!-- ====================================================================== -->
+## All error and warning messages in the Console now have a Search Web button
+
+<!-- Title: All console errors and warnings now have a Search Web button -->
+<!-- Subtitle: You can now search for any of your console errors and warnings right from DevTools. -->
+
+The **Search for this message on the Web** button is now available on all errors and warnings in the **Console**.  Previously, this button only appeared on some of the common errors and warnings.  Now the button has been added on the remaining errors and warnings.  The relevant error or warning string is used as the search string.
+
+For more information, see [Search for Console errors on the web](../09/devtools.md#search-for-console-errors-on-the-web).
+
+:::image type="content" source="../../media/2021/10/console-message-search-web-button.png" alt-text="All error and warning messages in the Console now have a Search Web button." lightbox="../../media/2021/10/console-message-search-web-button.png":::
+
+
+<!-- ====================================================================== -->
+## Improved keyboard access for defining User-Agent Client Hints
+
+<!-- Title: When adding a custom device to emulate in DevTools, specify user agent client hints more easily -->
+<!-- Subtitle: We've improved keyboard access when navigating to user agent clients from Settings. -->
```

Comment:
Can we switch these around?

Reply:
<!-- Title: Improved keyboard access when navigating to User agent client hints in Settings -->
<!-- Subtitle: When adding a custom device to emulate in DevTools, you can now expand the User agent client hints section more easily. -->

Reply:
Title: Improved keyboard access when navigating to User agent client hints in Settings
Subtitle: When adding a custom device to emulate in DevTools, you can now expand the User agent client hints section more easily.